src/routes/users/fetch-users.e2e.test.js

```diff
@@ -0,0 +1,59 @@
+const { expect } = require('chai')
+const { fixtures, setupDB } = require('../../../test/utils')
+const fetchUsers = require('../../services/users/fetch-users')
+
+describe('GET /users', () => {    
+	describe('Success', () => {
+		let adminUser
+		let accessToken
+		beforeEach(async () =>{
+			await setupDB()
+			await fixtures.insertUser()
+			adminUser = await fixtures.insertUser({
+				role : 'admin'
+			}) 
+			const body = { id: adminUser.id, email: adminUser.email }
+			accessToken = fixtures.generateAccessToken(
+				{user : body}
+			)
+		})
+		it('should return 200 if users are fetched', async () => 
+			fixtures.api()
+				.get(`/api/v1/users`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(200)
+		)
+		it('should return the right number of users', async() =>{
+			const actualUsers = await fetchUsers()
+			fixtures.api()
+				.get(`/api/v1/users`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.then(res => {
+				 expect(res.body.length).eql(actualUsers.length)
```

Comment:
Building upon the earlier point,  by the time we reach this point, we know the number of users inserted so it can be hardcoded without doing any new fetch